packages/cashier/src/modules/deposit-fiat/deposit-fiat.tsx

```diff
@@ -1,11 +1,20 @@
 import React from 'react';
+import { observer, useStore } from '@deriv/stores';
 import { PageContainer } from '../../components/page-container';
 import { DepositFiatIframe } from './components';
 
-const DepositFiat: React.FC = () => (
-    <PageContainer>
-        <DepositFiatIframe />
-    </PageContainer>
-);
+const DepositFiat: React.FC = observer(() => {
+    const { traders_hub } = useStore();
+    const { is_low_risk_cr_eu_real } = traders_hub;
+
+    return (
+        <PageContainer
```

Comment:
@farzin-deriv Isn’t PageContainer specific to cashier and our intention with modules is that it can be imported and used anywhere? Shouldn't PageContainer go a level up?
If I remember correctly, that’s what you had done initially right?
Something like,
`CashierLockedChecker`
...|_ `PageContainer`
.....|_ `Deposit`
........|_ `DepositFiat`
...........|_ `DepositFiatIframe`
Any reason for changing this now?

Reply:
@nijil-deriv Not really, `PageContainer` is a dumb layout component, It's only responsible for adding style and giving us 3 responsive columns to match the design system.

Later we will be moving this component to `@deriv/components` and naming it something more meaningful maybe so other apps can also use it, But currently it's in `@deriv/cashier` until it's more mature to handle all the cases. For example, in the discussion we had with the design team to always keep the right sidebar space even if it's empty, We need to first figure out all these cases and then once we have a generic component we can move it to `@deriv/components`.

Basically `PageContainer` will be wrapping the content of each route, But since we have it in `@deriv/cashier` for now and we have some general status checks for cashier in `CashierLockedChecker` I used it inside the `PageContainer` to avoid duplication until all the pages are using this `PageContainer` so then as @aum-deriv suggested before we would make the `CashierLockedChecker` into a module itself and move `PageContainer` outside.

But since the design team wants a specific behavior for `PageContainer`, We can't do this anymore :( So need to repeat `CashierLockedChecker` same as `PageContainer` for each page for now.

To answer the question about modules, We need to think of them more specifically on what they do, `DepositFiatModule` is not responsible for also checking the cashier statuses or asking for verify email or things like that, That would be a different module which @aum-deriv suggested before, But we need to go step by step so we can figure out the needs first, We don't know how `CashierLockedChecker` should transform to a module and what should it exactly do yet.

So long but I hope it answers the question 😅

Reply:
Okay, even if we remove `CashierLockedChecker` from the equation; `DepositFiatModule` shouldn't have `PageContainer` in it right, since it also controls `Breadcrumb`?

Reply:
@nijil-deriv Yes, Because a module is not necessarily a page, Imagine a dashboard page where we have deposit fiat and crypto side by side like widgets. That's why `PageContainer` will become dumb and move to `@deriv/components` in the future.